### PR TITLE
CORE-1722 merge devnet-algo-pg into devnet-algo

### PR DESCRIPTION
**Pull request by erer1243**

### CODE REVIEW

File: scripts/devnet-algo/docker-compose.yaml [hunk 1]
```diff
@@ -2,20 +2,7 @@ version: "3"
...
-    environment:
-      - POSTGRES_HOST=devnet-algo-pg
-      - POSTGRES_USER=algogrand
-      - POSTGRES_PASSWORD=indexer
-      - POSTGRES_DB=pgdb
     ports:
       - "4180:4180"
       - "8980:8980"
       - "9392:9392"
-  devnet-algo-pg:
```
Comment (jeapostrophe) [inline scripts/devnet-algo/docker-compose.yaml:16]: You should remove them from hs/app/reach/embed/docker/*.yml too

**Review by jeapostrophe — CHANGES_REQUESTED** [review]
Nice! Can't wait to see the CI win

File: scripts/devnet-algo/docker-compose.yaml (hunk 1 above)
Comment (erer1243) [reply scripts/devnet-algo/docker-compose.yaml:16]: Thanksf

**Review by jeapostrophe — COMMENTED** [review]
This looks fine to me, but I'd like @mattaudesse to review this part

**Review by mattaudesse — APPROVED** [review]
Looks good to me apart from these little suggestions.

**Review by jeapostrophe — APPROVED** [review]

**Comment by jeapostrophe** [comment]
The CI stalled, so I rebased it to make it go again, we'll see if that helps

**Pull request merged**